cypress/e2e/views/applicationinventory.view.ts

```diff
@@ -46,7 +46,7 @@ export const importStatus = "Status";
 export const selectBox = "input[type=checkbox]";
 export const tags = "span";
 export const user = "User";
-export const kebabMenu = "#row-actions";
+export const kebabMenu = "#action";
```

Comment:
This is not there in release-0.7 as well . 

https://github.com/konveyor/tackle-ui-tests/blob/release-0.7/cypress/e2e/views/applicationinventory.view.ts#L49

May be devs have merged some PRs for release-0.7 but not in main .